torch/_inductor/fx_passes/quantization.py

```diff
@@ -756,30 +787,42 @@ def _register_quantization_lowerings():
     _register_quantization_cat()
 
 
-def _is_valid_dequant_promotion_pattern(match):
-    mul_node = match.output_node()
-    sub_node = mul_node.args[0]
-    to_fp32_node = sub_node.args[0]
-    if (
-        mul_node.target is aten.mul.Tensor
-        and sub_node.target is aten.sub.Tensor
-        and to_fp32_node.target is prims.convert_element_type.default
-        and len(list(mul_node.users)) > 1
-    ):
-        # dequant pattern has more than 1 users to be promoted
-        return True
-    return False
+def _is_valid_dequant_promotion_pattern(dtype=torch.float32):
```

Comment:
could you add some docs to explain dequant_promotion

Reply:
Thanks for the suggestions. Docs added as in https://github.com/pytorch/pytorch/pull/112550#discussion_r1388459772